### PR TITLE
[docs] Add region and use of SSM decryption to Terraform docs

### PR DESCRIPTION
**Pull request by webvictim**
Adds region and decryption to AWS SSM commands as appropriate.

Backports required:
- `branch/v7`
- `branch/v8`

### CODE REVIEW

File: docs/pages/setup/deployments/aws-terraform.mdx [hunk 1]
```diff
@@ -714,7 +714,7 @@ To add new nodes/EC2 servers that you can "SSH into" you'll need to:
 You can use this command to get the CA pin hash for your Teleport cluster:
 
 ```code
-$ aws ssm get-parameter --name "/teleport/${TF_VAR_cluster_name}/ca-pin-hash" --query "Parameter.Value" --output text
+$ aws ssm get-parameter --region ${TF_VAR_REGION} --name "/teleport/${TF_VAR_cluster_name}/ca-pin-hash" --query "Parameter.Value" --output text
```
Comment (russjones) [inline docs/pages/setup/deployments/aws-terraform.mdx:717]: Why is `TF_VAR_REGION` all caps but `TF_VAR_cluster_name` mixed?

Comment (webvictim) [reply docs/pages/setup/deployments/aws-terraform.mdx]: Good catch, fixed

**Comment by webvictim** [comment]
Bump @russjones @r0mant

**Review by russjones — APPROVED** [review]

**Review by Joerger — APPROVED** [review]

**Comment by webvictim** [comment]
Backport to `branch/v7` is no longer required as those docs are not current.

**Pull request merged**